var/spack/repos/builtin/packages/pocl/package.py

```diff
@@ -21,7 +21,7 @@ class Pocl(CMakePackage):
 
     license("MIT")
 
-    version("master", branch="master")
+    version("master", branch="main")
```

Comment:
Should we just call it `main`?

Reply:
Or both, with `master` deprecated. See e.g. https://github.com/spack/spack/blob/0619a8bd4f2238d87b22779b30b6b923d83eb846/var/spack/repos/builtin/packages/acts/package.py#L42

Reply:
Done

Reply:
A lot of unrelated commits have snuck in. Can you remove them?